EIPS/eip-7870.md

```diff
@@ -0,0 +1,146 @@
+- **On Endurance (TBW)**  
+  - Running a node involves frequent writes (e.g., database updates, logs). Ensure that the SSD’s Total Bytes Written (TBW) rating is sufficient for multi-year operation.
+- **Capacity Considerations**  
+  - As of January 2025, 2 TB can still work, but daily chain growth makes 4 TB more future-proof.  
+  - While EIP-4444 aims to reduce historical storage requirements, the timeline for EIP-4444 remains uncertain.
+
+### Memory
+
+- **Why 64 GB for Validators (Attester / Local Block Builder)?**
+  - Running an Ethereum validator (both EL & CL clients) can be memory-intensive, with state cache dominating RAM usage.
+  - Certain memory intensive client combinations have historically failed with 16 GB.  
+  - Preliminary benchmarks (e.g., [zk-STARK memory usage](https://hackmd.io/@han/bench-hash-in-snark)) suggest cryptographic operations can demand significant RAM.  
+  - Relative to total hardware costs, upgrading from 32 GB to 64 GB is a small price but can improve stability and is more future proof with regards to zk-STARKS.
+
+### CPU
+
+- **Single vs. Multi-thread Performance**
+  - Attesters and local block builders benefit from both high single-thread performance and solid multi-thread performance.  
+
+### Bandwidth
+
+#### Local Block Builders
+
+- **Recommended**: 100 Mbps download, 50 Mbps upload.
+- **Rationale**:  
+  - Distributing blocks is highly bandwidth-intensive.  
+  - If a builder cannot meet these speeds, they risk slower propagation and causing late blocks.  
+  - In extreme cases, a low-bandwidth node could propose partially full blocks or one that includes fewer blobs to mitigate slow broadcast.
+
+#### Attesters
+
+- **Recommended**: 50 Mbps download, 25 Mbps upload.
+- **Minimum tested**: 15 Mbps (as per [ethPandaOps](https://ethpandaops.io/) simulations where 40% of the network had Gigabit connections and the other 60% had 15 Mbps upload).
+  - However, real-world performance depends on peer network topology. A node with poor bandwidth could in theory quickly share data with a well-connected peer with good bandwidth which means that this peer could quickly seed the network.
+- **Rationale**:
+  - 25 Mbps was chosen as a buffer to account for these miscellaneous factors that are harder to predict.
+
+#### Validators Using mev-boost
+
+- **Recommended**: 50 Mbps download / 25 Mbps upload.
+- **Rationale**:
+  - Most MEV-relay interactions involve fetching bundles and block headers, which can be done within typical broadband speeds.
+  - While the local validator will also share the block with its peers, the relay will do the same which reduces the need for local bandwidth.
+  - There may however be cases where your validator will still build a local block, such as if no mev-relay responds or if the value of the mev reward is lower than the minimum configuration set in mev-boost. In these circumstances, the recommendations for **Local Block Builders** would be relevant.
```

Comment:
```suggestion
  - However, there may be cases where your validator will still build a local block, such as if no MEV-relay responds or if the value of the MEV reward is lower than the minimum configuration set in MEV-Boost. In these circumstances, the recommendations for **Local Block Builders** would be relevant.
```
* We're missing commas around "however" but it flows better if we move it to the front.
* We use `MEV-relay` in another point. For consistency, we should capitalize MEV in other places too.
  * The "B" in boost should be capitalized too. See: https://github.com/flashbots/mev-boost#what-is-mev-boost

Reply:
Btw, there are other instances of "mev" -> "MEV" which need to be updated.